packages/common/src/utils/url-generator.ts

```diff
@@ -183,11 +183,16 @@ export const optionsToParameterizedUrl = (options: Object) => {
 export const gitHubToSandboxUrl = (githubUrl: string) =>
   githubUrl.replace(gitHubPrefix, '/s/github').replace(dotGit, '');
 
-export const gitHubToSandboxBetaUrl = (githubUrl: string) =>
-  githubUrl
-    .replace(gitHubPrefix, '/github')
+export const gitHubToSandboxBetaUrl = (githubUrl: string) => {
+  const [owner, repo] = githubUrl
+    .replace(gitHubPrefix, '')
     .replace(dotGit, '')
-    .replace(/\/tree\//, '/');
+    .replace(/\/tree\//, '/')
+    .split('/')
+    .filter(Boolean);
```

Comment:
Trying to figure out why the filter is needed at the end, can there be duplicate slashes in the url?

Reply:
The initial URL looks like this `/owner/project`, which would split into `["", "owner", "project:]`, does it make sense?